cabot_alert_email/models.py

```diff
@@ -44,5 +44,5 @@ def send_alert(self, service, users, duty_officers):
             subject=subject,
             message=t.render(c),
             from_email='Cabot <%s>' % env.get('CABOT_FROM_EMAIL'),
-            recipient_list=emails,
+            recipient_list=list(emails),
```

Comment:
Not sure whether we need to convert to list, since a set should be iterable just fine...